pennylane_lightning/core/src/simulators/lightning_qubit/CMakeLists.txt

```diff
@@ -61,12 +61,12 @@ set_property(TARGET lightning_qubit PROPERTY POSITION_INDEPENDENT_CODE ON)
 ###############################################################################
 # Include subdirectories
 ###############################################################################
-set(COMPONENT_SUBDIRS   algorithms
+set(COMPONENT_SUBDIRS   utils
```

Comment:
Can `utils` be move to last in this last to be in alphabetical order?

Reply:
Done!